nx/source/services/nifm.c

```diff
@@ -81,6 +82,177 @@ Result nifmGetCurrentIpAddress(u32* out) {
     return rc;
 }
 
+Result nifmIsWirelessCommunicationEnabled(bool* out) {
+    IpcCommand c;
+    ipcInitialize(&c);
+
+    struct {
+        u64 magic;
+        u64 cmd_id;
+    } *raw;
+
+    raw = serviceIpcPrepareHeader(&g_nifmIGS, &c, sizeof(*raw));
+
+    raw->magic = SFCI_MAGIC;
+    raw->cmd_id = 17;
+
+    Result rc = serviceIpcDispatch(&g_nifmIGS);
+
+    if (R_SUCCEEDED(rc)) {
+        IpcParsedCommand r;
+
+        struct {
+            u64 magic;
+            u64 result;
+            u8 out;
+        } *resp;
+
+        serviceIpcParse(&g_nifmIGS, &r, sizeof(*resp));
+        resp = r.Raw;
+
+        rc = resp->result;
+        *out = resp->out;
```

Comment:
newline, and: `if (R_SUCCEEDED(rc) && out) *out = resp->out!=0;` (same below)

Reply:
ping ^

Reply:
Do you want the line on 112 removed?

Reply:
no https://github.com/switchbrew/libnx/pull/236/files#r250879994